README.md

```diff
@@ -936,4 +936,4 @@ It's a shame that Google, with their immense resources, power, and influence, do
 
 Hopefully this guide can serve as a starting point for those new to privacy, or be a good refresher for the experts.
 
-Good luck and stay safe!
\ No newline at end of file
+Good luck and stay safe!
```

Comment:
No idea why git thinks this is a change, looks identical to me

Reply:
I think because they changed from place in the file

Reply:
One doesn't have a newline at the end of the file, the other one has.